GOVERNANCE.md

```diff
@@ -63,7 +63,7 @@ The responsibilities of the Technical Committee are
 * coordinating event participation and communications regarding the Eiffel Community Open Source Projects
 
 Eiffel Community aims to operate as a consensus based community, if any decision requires a vote to move the Eiffel Community Open Source Projects forward, the members of the Technical Committee will vote on a one vote per member basis.
-Simple majority is required for any decision to be approved.
+Simple majority is required for any decision to be approved. When voting in a meeting, it is a simple majority of the voting members present.  When voting through email or other forms of asynchronous communication, it is a simple majority of all TSC voting members.
```

Comment:
What is 'TSC'? I assume you mean Technical Committee? I think we should avoid unnecessary abbreviations.

Reply:
TSC means "Technical Steering Committee" but we name it "Technical Committee" in Eiffel. We will need to document the role of the committee during bootstrap phase as part of the committee charter work to clarify it further. (will the committee do technical steering or just overseeing and so on.)